packages/smooth_app/lib/helpers/product_cards_helper.dart

```diff
@@ -11,14 +11,22 @@ import 'package:smooth_app/helpers/image_field_extension.dart';
 import 'package:smooth_app/helpers/ui_helpers.dart';
 import 'package:smooth_app/query/product_query.dart';
 
+String getProductNameAndBrands(
```

Comment:
```suggestion
Widget getProductTitle(
  final Product product,
  final AppLocalizations appLocalizations,
) => Text(
    getProductNameAndBrands(upToDateProduct, appLocalizations),
    overflow: TextOverflow.ellipsis,
    maxLines: 1,
  );

String getProductNameAndBrands(
```

Reply:
Named it `buildProductTitle`, so it's hopefully gives a hint, that it returns different type, compared to `getProductName` below.